dist/overflow-color.cjs.js

```diff
@@ -56,11 +56,9 @@ var checkScroll = function checkScroll() {
     requestAnimFrame(function () {
       var scrollHeight = document.body.scrollHeight;
       var innerHeight = window.innerHeight;
-      if (scrollHeight === innerHeight) {
-        setBgColor(bottomColor);
-      } else {
-        setBgColor(innerHeight - scrollHeight + 2 * lastScrollY < 0 ? topColor : bottomColor);
-      }
+
+      setBgColor(innerHeight - scrollHeight + 2 * lastScrollY > 0 ? bottomColor : topColor);
```

Comment:
Hey Igor! Like this version a lot better :-)

Given how hard it was for us to figure out what this math did, would you mind adding a quick comment in each place where it's used along the lines of:

```
// If we're more than halfway down the page, use the bottom color. Otherwise,
// use the top color.
```